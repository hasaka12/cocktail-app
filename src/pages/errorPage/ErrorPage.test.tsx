import { render } from "@testing-library/react";
import ErrorPage from "./ErrorPage";

describe("ErrorPage", () => {
  test("renders ErrorPage component", () => {
    const { getByText } = render(<ErrorPage />);
    const errorPageElement = getByText(/ErrorPage/i);
    expect(errorPageElement).toBeInTheDocument();
  });

  test("Renders correctly", () => {
    const { asFragment } = render(<ErrorPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
