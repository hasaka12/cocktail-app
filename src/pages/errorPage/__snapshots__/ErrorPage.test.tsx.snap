// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage Renders correctly 1`] = `
<DocumentFragment>
  <div>
    ErrorPage
  </div>
</DocumentFragment>
`;
